libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Login/SFLoginViewController.h

```diff
@@ -106,14 +103,28 @@
  */
 - (void)hideHostListView:(BOOL)animated;
 
-/** Determine whether or not the back button should be shown on navigation bar
+/** Factory Method to create the back button.
  */
-- (BOOL)shouldShowBackButton;
+- (UIBarButtonItem *_Nonnull)createBackButton;
```

Comment:
Nit: Can we use the `nonnull` preceding nomenclature here, as well as for the similar items below?  E.g.:
```obj-c
- (nonnull UIBarButtonItem *)createBackButton;
```